parsl/providers/slurm/slurm.py

```diff
@@ -59,6 +59,9 @@ class SlurmProvider(ClusterProvider, RepresentationMixin):
         String to prepend to the #SBATCH blocks in the submit script to the scheduler.
     worker_init : str
         Command to be run before starting a worker, such as 'module load Anaconda; source activate env'.
+    exclusive : bool (Default = True)
+        Requests nodes which are not shared with other running jobs. Executors such as HighThroughputExecutor
+        and ExtremeScaleExecutor assume that the nodes are exclusive.
```

Comment:
Based on our discussion, we should remove htex and exex from the docstring to avoid confusion.